packages/account/src/Configs/trading-assessment-config.js

```diff
@@ -0,0 +1,305 @@
+import { generateValidationFunction, getDefaultFields } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+
+export const trading_assessment_questions = [
+    {
+        question_text: 'Do you understand that you could potentially lose 100% of the money you use to trade?',
```

Comment:
Are we localizing ALL of these question texts?

Reply:
Resolved 🫡